### PR TITLE
Cosmetic tweaks

### PR DESCRIPTION
**Pull request by brendanheywood**

### CODE REVIEW

File: lang/en/tool_dataflows.php [hunk 1]
```diff
@@ -285,14 +285,13 @@
...
 $string['connector_curl:destination'] = 'File / Response destination';
-$string['connector_curl:destination_help'] = 'Should this field be left blank,
-response will be stored in step definition to be reused subsequently by other steps.';
-$string['connector_curl:headers'] = 'Headers';
-$string['connector_curl:method'] = 'HTTP request method';
+$string['connector_curl:destination_help'] = 'If set then the response body will be saved to this path. If blank then the response body will available in the step variable: ...response.result';
+$string['connector_curl:headers'] = 'HTTP request headers';
 $string['connector_curl:rawpostdata'] = 'Raw post data';
 $string['connector_curl:sideeffects'] = 'Does this request have side effects?';
+$string['connector_curl:sideeffects_help'] = 'Most read requests done via http GET do not have side effects ie they change state on the remote server, and generally any POST of PUT does have side effects. Curl calls with side effects are not actually executed in dry run mode.';
```
Comment (petersistrom) [inline lang/en/tool_dataflows.php:294]: Should it say 'and generally any POST or PUT'?

File: lang/en/tool_dataflows.php [hunk 2]
```diff
@@ -285,14 +285,13 @@
 $string['connector_s3:source_is_a_directory'] = 'The source path is a directory but a file path is expected.';
 
 // Connector cURL.
-$string['connector_curl:curl'] = 'Client URL';
+$string['connector_curl:curl'] = 'URL';
 $string['connector_curl:destination'] = 'File / Response destination';
-$string['connector_curl:destination_help'] = 'Should this field be left blank,
-response will be stored in step definition to be reused subsequently by other steps.';
-$string['connector_curl:headers'] = 'Headers';
-$string['connector_curl:method'] = 'HTTP request method';
```
Comment (petersistrom) [inline lang/en/tool_dataflows.php:293]: This is string called in connector_curl.php - https://github.com/catalyst/moodle-tool_dataflows/pull/312/files#diff-cf8d12918f4729abaeec73d8b5c28ddb1d00c306ee84d79ce193c5ea0dc8308aR74

File: lang/en/tool_dataflows.php [hunk 3]
```diff
@@ -285,14 +285,13 @@
...
+$string['connector_curl:destination_help'] = 'If set then the response body will be saved to this path. If blank then the response body will available in the step variable: ...response.result';
+$string['connector_curl:headers'] = 'HTTP request headers';
 $string['connector_curl:rawpostdata'] = 'Raw post data';
 $string['connector_curl:sideeffects'] = 'Does this request have side effects?';
+$string['connector_curl:sideeffects_help'] = 'Most read requests done via http GET do not have side effects ie they change state on the remote server, and generally any POST or PUT does have side effects. Curl calls with side effects are not actually executed in dry run mode.';
```
Comment (keevan) [inline lang/en/tool_dataflows.php:294]:
```suggestion
$string['connector_curl:sideeffects_help'] = 'Most read requests done via http GET do not have side effects ie they change state on the remote server, but POST and PUT generally do have side effects. Curl calls with side effects are not actually executed in dry run mode.';
```

File: classes/local/step/connector_curl.php [hunk 4]
```diff
@@ -68,21 +68,37 @@ public function form_add_custom_inputs(\MoodleQuickForm &$mform) {
         ];
         $ex['json'] = \html_writer::nonempty_tag('pre', json_encode($jsonexample, JSON_PRETTY_PRINT));
         $ex['yaml'] = '<pre>header-key: header value<br>another-key: 1234</pre>';
-        $mform->addElement('text', 'config_curl', get_string('connector_curl:curl', 'tool_dataflows'),
-                ['cols' => 50, 'rows' => 7]);
-        $mform->addElement('text', 'config_destination', get_string('connector_curl:destination', 'tool_dataflows'));
-        $mform->addHelpButton('config_destination', 'connector_curl:destination', 'tool_dataflows');
+
+
```
Comment (petersistrom) [inline classes/local/step/connector_curl.php:72]: Functions must not contain multiple empty lines in a row; found 2 empty lines

File: classes/local/step/connector_curl.php [hunk 5]
```diff
@@ -68,21 +68,37 @@ public function form_add_custom_inputs(\MoodleQuickForm &$mform) {
...
+        $urlarray = [];
+        $urlarray[] =& $mform->createElement('select', 'config_method', '', [
+            'get'    => 'GET',
+            'post'   => 'POST',
+            'head'   => 'HEAD',
+            'patch'  => 'PATCH',
+            'put'    => 'PUT',
+        ]);
+        $urlarray[] =& $mform->createElement('text', 'config_curl', '') ;
```
Comment (petersistrom) [inline classes/local/step/connector_curl.php:81]: Space found before semicolon; expected ");" but found ") ;

File: classes/local/step/reader_sql.php [hunk 6]
```diff
@@ -267,13 +267,15 @@ public function validate_config($config) {
...
+  SELECT id,
+         username
+    FROM {user}
+[[ WHERE id > \${{countervalue}} ]]
+ORDER BY id ASC
+   LIMIT 10";
         $sqlexamples = \html_writer::tag('pre', trim($sqlexamples, " \t\r\0\x0B"));
-        $mform->addElement('textarea', 'config_sql', get_string('reader_sql:sql', 'tool_dataflows'), ['cols' => 50, 'rows' => 7]);
+
+        $mform->addElement('textarea', 'config_sql', get_string('reader_sql:sql', 'tool_dataflows'), ['cols' => 50, 'rows' => 7, 'style' => 'font: 87.5% monospace;']);
```
Comment (petersistrom) [inline classes/local/step/reader_sql.php:278]: Line exceeds 132 characters; contains 167 characters.

**Review by keevan — CHANGES_REQUESTED** [review]
You will need to bump the version.php because:
- lang strings have been updated

**Pull request merged**